src/composables/queries/useUserPoolsQuery.ts

```diff
@@ -145,13 +147,84 @@ export default function useUserPoolsQuery(
     );
     await injectTokens(tokens);
     await forChange(dynamicDataLoading, false);
+
     const decoratedPools = await balancerSubgraphService.pools.decorate(
       pools,
       '24h',
       prices.value,
       currency.value
     );
 
+    for (let i = 0; i < decoratedPools.length; i++) {
```

Comment:
This needs to happen *after* pool decoration otherwise it gets overwritten.

Reply:
Realise this is adding tech debt essentially. I'm hoping to completely refactor pool fetching in the new year.